ipaclient/install/ipa_client_automount.py

```diff
@@ -329,13 +329,6 @@ def uninstall(fstore, statestore):
     ]
     STATES = ['autofs', 'rpcidmapd', 'rpcgssd']
 
-    # automount only touches /etc/nsswitch.conf if LDAP is
-    # used. Don't restore it otherwise.
-    if statestore.get_state('authconfig', 'sssd') or (
-        statestore.get_state('authselect', 'profile') == 'sssd'
-    ):
-        RESTORE_FILES.remove(paths.NSSWITCH_CONF)
-
```

Comment:
I figured out why I added this. Try this sequence:

set automember to files in /etc/nssswitch.conf
ipa-client-install
grep automount /etc/nsswitch.conf (should be sss files)
ipa-client-automount --no-sssd
grep automount /etc/nsswitch.conf (should be files ldap)
ipa-client-automount --uninstall -U
grep automount /etc/nsswitch.conf will be files, the original state

Reply:
Fixed, thank you!